integration_tests/testsuite/test_runtime_image.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import subprocess
+import sys
+
+import deploy_app
+import test_util
+
+
+def _deploy_app(appdir):
+    try:
+        # change to app directory (and remember original directory)
+        owd = os.getcwd()
+        os.chdir(appdir)
+
+        deployed_version = test_util.generate_version()
+
+        # TODO: once sdk driver is published, use it here
+        deploy_command = ['gcloud', 'app', 'deploy',
+                          '--version', deployed_version, '-q']
+
+        subprocess.check_output(deploy_command)
+
+        return deployed_version
+    except subprocess.CalledProcessError as cpe:
+        logging.error('Error encountered when deploying application! %s',
+                      cpe.output)
+        sys.exit(1)
+
+    finally:
+        deploy_app._cleanup(appdir)
+        os.chdir(owd)
+
+
+def _main(appdir):
```

Comment:
the `_` in front of a method typically denotes it as being private (though this isn't really enforced by the interpreter), so no need for it on `main()`